README.md

```diff
@@ -1,8 +1,19 @@
-# Security Force Monitor: Backend [![Build Status](https://travis-ci.org/security-force-monitor/sfm-cms.svg?branch=master)](https://travis-ci.org/security-force-monitor/sfm-cms)
+# Who Was In Command [![Build Status](https://github.com/security-force-monitor/sfm-cms/actions/workflows/main.yml/badge.svg?branch=master)](https://github.com/security-force-monitor/sfm-cms/actions?query=branch%3Amaster)
 
-## OS Level dependencies
+- [Dependencies](#dependencies)
+- [Development](#development)
+- [Translations](#translations)
+- [Tests](#tests)
+- [Using the Google Drive import script](#using-the-google-drive-import-script)
 
-* Docker and Docker Compose
+**See also:**
+
+- [Who Was In Command: Getting started with development](DEVELOPMENT.md)
+- [Security Force Monitor: Research Handbook](https://help.securityforcemonitor.org/en/latest/index.html)
+
+## Dependencies
```

Comment:
Derek has already tested the revised local development instructions and confirmed they result in a running app!